app/policies/feed_token_policy.rb

```diff
@@ -0,0 +1,9 @@
+class FeedTokenPolicy < ApplicationPolicy
+  def update?
```

Comment:
I think you'll eventually need to chain `create?` and `destroy?` to this update method.

And you'll also need to add a `show?` to the `FeedPolicy` - it seems like right now, I can change the `Podcast.update(prx_account_uri: "/api/v1/accounts/999999")`, and the page still loads for me.